src/Behat/Behat/Output/Node/EventListener/AST/OutlineListener.php

```diff
@@ -136,10 +137,12 @@ private function printExampleHeaderOnBeforeExampleEvent(Formatter $formatter, Ev
             return;
         }
 
-        $this->example = $event->getScenario();
-
         $this->exampleSetupPrinter->printSetup($formatter, $event->getSetup());
-        $this->examplePrinter->printHeader($formatter, $event->getFeature(), $this->example);
+
+        $this->example = $event->getScenario();
```

Comment:
The structure of the way the event listeners is registered is somewhat complex, but I **think** the assumption is that events only reach this listener if we're in the course of running an Outline and therefore the scenario **should** be an ExampleNode?

If so, getting an unexpected type of ScenarioNode here would suggest either there's a problem with our event binding, or with our assumptions about how the nodes / events are structured. 

In which case it might be better / safer to throw an exception (and hopefully fail CI) rather than silently not print anything? This would be the intentional version of the current behaviour where it will hit a TypeError.